packages/server/src/user/user.service.ts

```diff
@@ -0,0 +1,164 @@
+import { Injectable } from '@nestjs/common';
+import { Prisma, USER } from '@prisma/client';
+import { PrismaService } from '../prisma/prisma.service';
+
+@Injectable()
+export class UserService {
+  constructor(private prisma: PrismaService) {}
+
+  // TO BE REMOVED
```

Comment:
A common convention for comments that relate for things to be done later is to add `TODO:` before them. Its a common enough convention that there are a lot of tools out there for finding TODOs. Plus it makes it harder to miss left over incomplete tasks since you can then search the project for TODOs

An example would be
```
// TODO: Remove temporary users
```

Reply:
Not sure these conversations are ready to be marked as resolved. I still think it would be best to use the `TODO` convention to ensure the correct things are handled in the future.